tsl/src/nodes/decompress_chunk/decompress_chunk.c

```diff
@@ -84,7 +84,7 @@ static EquivalenceClass *
 append_ec_for_seqnum(PlannerInfo *root, CompressionInfo *info, SortInfo *sort_info, Var *var,
 					 Oid sortop, bool nulls_first)
 {
-	MemoryContext oldcontext = MemoryContextSwitchTo(root->planner_cxt);
+	ExplicitMemoryContext oldcontext = MemoryContextSwitchTo(root->planner_cxt);
```

Comment:
This one can also be switched to use the macro I think?

Reply:
Yes, but it was a long streak of code and I was thinking that y'all did not want to use the macro here (that's what the `ExplicitMemoryContext` is used for). I'm fine with switching this.

Reply:
Using macro here as well.